app/controllers/capture_controller.rb

```diff
@@ -17,9 +17,10 @@ def index
       elsif current_user.calibrations.count > 0
         @calibration = current_user.last_calibration
       end
-      @calibrations = Spectrum.where(calibrated: true, user_id: current_user.id)
-      @calibrations << @calibration if @calibration
-      @calibrations.uniq!
```

Comment:
Workaround for making all elements in @ calibrations unique because @ calibrations.uniq was giving immutable relation error